docs/user-guide/utils.rst

```diff
@@ -41,7 +41,7 @@ Gammapy these are also the recommended format and time scale.
 
 .. warning::
 
-   Actually what's written here is not true. In CTA it hasn't been decided if
+   Actually what's written here is not true. In CTAO, it hasn't been decided if
```

Comment:
I don't understand why ` Actually what's written here is not true.` I would simply write `CTAO hasn't decided whether`

Reply:
I comment this part that is partially a relic of old times